tests/test_isrFunctions.py

```diff
@@ -104,6 +196,166 @@ def test_makeThresholdMask(self):
 
         self.assertEqual(len(defectList), 1)
 
+    def test_ITLEdgeBleedMask(self):
+        """Expect number of masked pixels according to edge bleed masking.
+        """
+        detector = TestITLDetector()
+        exposure = TestITLExposure(detector.getBBox())
+        exposure.setDetector(detector)
+        exposure.mask.array[:, :] = 0
+        satMaskBit = exposure.mask.getPlaneBitMask('SAT')
+
+        saturationLevel = 12000.
+        exposure.fillSaturationMetadata(saturationLevel)
+
+        badAmpDict = {}
+        for amp in exposure.getDetector():
+            if amp.getName() == 'C10':
+                badAmpDict[amp.getName()] = True
+            else:
+                badAmpDict[amp.getName()] = False
+
+        # Case 1:
```

Comment:
The three cases are three separate tests, with separate checks that they work.  I know from the image that these work, but the tests would succeed even if only one of the test cases worked.